pythoncz/static/data/business.geojson

```diff
@@ -917,5 +917,17 @@
                 "coordinates": [[14.4525094, 50.0924147]]
             }
         }
+         {
+            "type": "Feature",
+            "properties": {
+                "name": "BoxtrapSecurity",
+                "url": "https://www.boxtrapsecurity.com/",
+                "company": true
+            },
+            "geometry": {
+                "type": "MultiPoint",
+                "coordinates": [[50.0823494, 14.4220517]]
```

Comment:
Unfortunately, GeoJSON has latitude and longitude swapped compared to the rest of the world, so this would point to Yemen. Could you please put it in order `14.xxx, 15.xxx`? Thanks!

Reply:
I meant `14.xxx, 50.xxx` 😀